src/wallet/wallet.cpp

```diff
@@ -1896,20 +1896,25 @@ void CWallet::ReacceptWalletTransactions(interfaces::Chain::Lock& locked_chain)
 
 bool CWalletTx::RelayWalletTransaction(interfaces::Chain::Lock& locked_chain)
 {
-    assert(pwallet->GetBroadcastTransactions());
-    if (!IsCoinBase() && !isAbandoned() && GetDepthInMainChain(locked_chain) == 0)
-    {
-        CValidationState state;
-        /* GetDepthInMainChain already catches known conflicts. */
```

Comment:
Could keep this comment?

Reply:
This comment is replaced by the `Don't relay conflicted or already confirmed transactions` comment above the call to `GetDepthInMainChain()`.